document/execution/instructions.rst

```diff
@@ -77,9 +89,9 @@ Numeric Instructions
 .. math::
    \begin{array}{lcl@{\qquad}l}
    (t\K{.}\CONST~c_1)~(t\K{.}\CONST~c_2)~t\K{.}\binop &\stepto& (t\K{.}\CONST~c)
```

Comment:
No, in this notation the stack grows to the right, the order in which the instructions would be executed. Added a respective clarification to the conventions section.